packages/x-data-grid/src/constants/localeTextConstants.ts

```diff
@@ -122,7 +122,8 @@ export const GRID_DEFAULT_LOCALE_TEXT: GridLocaleText = {
   filterValueFalse: 'false',
 
   // Column menu text
-  columnMenuLabel: 'Menu',
+  columnMenuLabel: 'Column options',
+  columnMenuAriaLabel: (columnName: string) => `${columnName} column options`,
```

Comment:
Since we consistently call it `column menu` in the docs ([example](https://deploy-preview-16796--material-ui-x.netlify.app/x/react-data-grid/column-menu/)), probably this would be more relatable?
```suggestion
  columnMenuLabel: 'Column menu',
  columnMenuAriaLabel: (columnName: string) => `${columnName} column menu`,
```

Reply:
> Since we consistently call it column menu in the docs ([example](https://deploy-preview-16796--material-ui-x.netlify.app/x/react-data-grid/column-menu/))

I think this is a strong argument to refer to it as a menu rather than options. I'm going to leave the `columnMenuLabel` string as it is currently in master: "Menu". There is enough of a visual cue that it is a menu for columns given where the button is located.

The only change then will be adding a separate locale string for the `aria-label` which I will make `${columnName} column menu` as you have suggested.